app/staging/domain/cases-summary.js

```diff
@@ -15,7 +15,9 @@ class CaseSummary {
     licenseTiers,
     custodyTiers,
     comIn1st16Weeks,
-    licIn1st16Weeks
+    licIn1st16Weeks,
+    armsCommunityCases,
+    armsLicenseCases
```

Comment:
So these are now pulled from staging and added to the cases-summary?

Reply:
Correct